api/src/main/resources/workbench.yaml

```diff
@@ -960,6 +960,48 @@ paths:
           description: A list of resources order by last access time desc order
           schema:
             $ref: "#/definitions/RecentResourceResponse"
+
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/recent-resource/{notebook}/update-notebook:
```

Comment:
Not sure if you already addressed my other comment that you said you'd missed, but per that comment I would change this to just "update", and put the notebookname into the request